.circleci/config.yml

```diff
@@ -277,12 +277,16 @@ jobs:
               echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
               VERSION=$(cat lerna.json | jq .version --raw-output)
               ./node_modules/.bin/lerna publish --skip-git --repo-version $VERSION --yes --force-publish=* --npm-client=npm
+            elif [[ "$CIRCLE_BRANCH" =~ '^v\d+\.\d+' ]]; then 
```

Comment:
instead of `elif` here shouldn't we just add an `or` in the main if similar to line 241?

Reply:
No we can't do that because we want to set the `--npm-tag` if we're doing a backport. It defaults to `latest` which would mean if we publish an older version and someone does an `npm install` they'll get that version instead of the actual latest version. So we want to specify a tag in that case.

Reply:
ok